dvc/command/data_sync.py

```diff
@@ -7,6 +7,10 @@
 
 
 class CmdDataBase(CmdBase):
+    def __init__(self, args):
+        self.UP_TO_DATE_MSG = "Everything is up-to-date."
+        super().__init__(args)
```

Comment:
We support py2 (at least for now :wink: ), so you'll need to use `super(CmdDataBase, self)` here instead. Or, you could avoid it at all by simply defining UP_TO_DATE_MSG in the class itself:
```
class CmdDataBase(CmdBase):
    UP_TO_DATE_MSG = "Everything is up to date."
    def run(self, target):
...
```

Also, notice that there is no need for `-` in `Everything is up to date.`, since it is after a noun, unless I'm missing something.

Reply:
https://github.com/iterative/dvc/pull/1620/commits/c12fae0f1dee5d58349582eea8a133ab7cbc768e